variables.tf

```diff
@@ -82,10 +82,16 @@ variable "website" {
   default     = {}
 }
 
-variable "cors_rule" {
-  description = "Map containing a rule of Cross-Origin Resource Sharing."
-  type        = any # should be `map`, but it produces an error "all map elements must have the same type"
-  default     = {}
+variable "cors_rules" {
+  description = "List containing rules for Cross-Origin Resource Sharing."
+  type        = list(object({
```

Comment:
lets use
```hcl
type = list(any)
```

instead. if we spell out each attribute like below, this breaks the optional arguments and becomes very strict

Reply:
@bryantbiggs changes have been made.

Personally I'm a fan of the strict, however, I also understand the desire for optional arguments :)